### ISSUE TITLE
Gantt chart with column tooltipText and cursor broken in 4.1.8?

### ISSUE THREAD

**Issue by bmampaey** [amcharts/amcharts4#988]
I tried to [post](https://github.com/amcharts/amcharts4/issues/668#issuecomment-464353059) about this on issue #668 , but now I think it may be a bug, so I open a new issue.

Since 4.1.8, adding a cursor to a Gantt chart breaks the columns tooltip (it does not appear, or very briefly), even if I add `series1.cursorTooltipEnabled = false;`

To demonstrate, here is the codepen of the "Gantt Chart" from the Demos where I only added the following 2 lines at the bottom https://codepen.io/Benjamin-Mampaey/pen/KJrZeO

```
// Add cursor
chart.cursor = new am4charts.XYCursor();

// Set cursorTooltipEnabled to false on series
series1.cursorTooltipEnabled = false;
```
The same codepen where I force the library to 4.1.7 seems to work, with or without disabling the cursor tooltip https://codepen.io/Benjamin-Mampaey/pen/KJJrmN

Note that the "Floating Bar Chart" demo does not have the problem, columns tooltip are shown as expected even when a cursor is added  https://codepen.io/Benjamin-Mampaey/pen/WPYdyW

**Comment by martynasma** [comment]
OK, so to fix this do:

```
series1.cursorTooltipEnabled = false;
```

Because without it it tries to display a cursor tooltip both as hover and as category.

**Comment by bmampaey** [comment]
No it doesn't work anymore, that's why I opened an issue.

As you can see in the example codepen I provided https://codepen.io/Benjamin-Mampaey/pen/KJrZeO , I have set `series1.cursorTooltipEnabled = false;` but the tooltips don't show.

But **it used to work** for version 4.1.7 and below.

**Comment by bmampaey** [comment]
The problem was solved in 4.1.10 (with or without `series1.cursorTooltipEnabled = false;`) , so I close the issue.

https://codepen.io/Benjamin-Mampaey/pen/eXpdaa
